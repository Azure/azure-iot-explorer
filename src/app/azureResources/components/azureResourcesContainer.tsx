import * as React from 'react';
import { RouteComponentProps } from 'react-router-dom';
import { ConnectionStringsViewContainer } from '../../connectionStrings/components/connectionStringsViewContainer';

export const AzureResourcesContainer: React.FC<RouteComponentProps> = props => {
       return <ConnectionStringsViewContainer {...props} />;
};
