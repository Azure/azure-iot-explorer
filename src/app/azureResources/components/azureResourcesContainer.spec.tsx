/***********************************************************
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License
 **********************************************************/
import * as React from 'react';
import { shallow } from 'enzyme';
import { AzureResourcesContainer } from './azureResourcesContainer';

describe('azureResourcesContainer', () => {
    it('matches snapshot', () => {

        const routerprops = {
            history: jest.fn() as any, // tslint:disable-line:no-any
            location: jest.fn() as any, // tslint:disable-line:no-any
            match: {
                params: {
                    hostName: 'hostName'
                },
                url: 'currentUrl',
            } as any // tslint:disable-line:no-any
        };
        const wrapper = shallow(<AzureResourcesContainer {...routerprops} />);
        expect(wrapper).toMatchSnapshot();
    });
});
