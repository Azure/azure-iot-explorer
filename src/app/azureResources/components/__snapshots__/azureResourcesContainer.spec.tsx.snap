// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`azureResourcesContainer matches snapshot 1`] = `
<Component
  history={[MockFunction]}
  location={[MockFunction]}
  match={
    Object {
      "params": Object {
        "hostName": "hostName",
      },
      "url": "currentUrl",
    }
  }
/>
`;
